bayesian/api/api_v2.py

```diff
@@ -39,6 +39,7 @@
     get_known_unknown_pkgs, add_unknown_pkg_info, get_batch_ca_data
 from bayesian.utils import (get_system_version,
                             server_create_component_bookkeeping,
```

Comment:
Is this component being used?

Reply:
when CA get call is made that is used to store data in api_requests table.

Reply:
I dont think we need to track that anymore.

Reply:
I will modify accordingly.